modules/publinkIdSystem.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * This module adds the PublinkId to the User ID module
+ * The {@link module:modules/userId} module is required
+ * @module modules/conversantIdSystem
+ * @requires module:modules/userId
+ */
+
+import {submodule} from '../src/hook.js';
+import {getStorageManager} from '../src/storageManager.js';
+import {ajax} from '../src/ajax.js';
+import * as utils from '../src/utils.js';
+import {uspDataHandler} from '../src/adapterManager.js';
+
+const MODULE_NAME = 'publink';
+const GVLID = 24;
+const SERVER_URL = utils.parseUrl('https://proc.ad.cpe.dotomi.com/cvx/client/direct/publink');
```

Comment:
Move the call to parseUrl to publinkIdUrl function so it doesn't take up CPU every time prebid loads.

Reply:
Also the path is /cvx/client/**sync**/publink